src/Skeletor/Packages/PackageInterface.php

```diff
@@ -8,9 +8,9 @@ public function getInstallSlug();
     public function setInstallSlug(string $installSlug);
     public function getName();
     public function setName(string $name);
-    public function getVersion();
+    public function getVersion(bool $allowEmpty);
```

Comment:
In Package the signature has a default value of true, don't we need it here too?